cbindings/pedersen.t.cc

```diff
@@ -54,6 +60,20 @@ static std::vector<c21t::element_p3> compute_random_generators(uint64_t seq_leng
   return generators;
 }
 
+//--------------------------------------------------------------------------------------------------
+// get_bls12_381_g1_generators
+//--------------------------------------------------------------------------------------------------
+static std::vector<cg1t::element_p2> get_bls12_381_g1_generators(uint64_t seq_length,
+                                                                 uint64_t offset) {
+  std::vector<cg1t::element_p2> generators(seq_length);
+
+  for (uint64_t i = 0; i < seq_length; ++i) {
```

Comment:
Can you add a note here that this s a placeholder until we get a method to make arbitrary generators?

Reply:
Good idea. Added.